packages/core/src/components/private/FormInput.tsx

```diff
@@ -148,4 +148,5 @@ FormInput.propTypes = {
   small: sizingProp,
 };
 
-export default FormInput;
+// Proofreader crashes in non-latin languages unless this component is memoized
```

Comment:
why does it crash?

Reply:
Look at the screenshots. It doesn't collapse CJK characters.